sled-agent/src/rack_setup/service.rs

```diff
@@ -291,6 +295,18 @@ impl RackSetupService {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, Default)]
+struct RssStartedMarker {}
+
+impl Ledgerable for RssStartedMarker {
+    fn is_newer_than(&self, _other: &Self) -> bool {
+        true
+    }
+    fn generation_bump(&mut self) {}
+}
+
+const RSS_STARTED_FILENAME: &str = "rss-started.marker";
+
 #[derive(Clone, Serialize, Deserialize, Default)]
 struct RssCompleteMarker {}
```

Comment:
Noting this for myself and others: The "RSS completed" marker is not changing with this PR, so systems which have fully deployed with RSS already will be unimpacted by this change. (This is good)

Reply:
Yep, done on purpose. I thought about adding a comment in code but didn't end up doing it. Happy to if you think it will help though.